src/wagmiConfig.ts

```diff
@@ -26,9 +13,46 @@ const transports = config.alchemyApiKey
       [polygon.id]: http(''),
     };
 
-export const wagmiConfig = createConfig({
-  chains: [polygon],
-  connectors,
+// 2. Create a metadata object - optional
+const metadata = {
+  name: 'Vortex',
+  description: 'Vortex',
+  url: 'https://app.vortexfinance.co', // origin must match your domain & subdomain
+  icons: ['https://avatars.githubusercontent.com/u/179229932'],
```

Comment:
What is the significance of this?

Reply:
Nice catch, I forgot about this. If you specify the metadata, the wallet app can show additional info when the WalletConnect connection is requested. I removed the icon but while `icons` cannot be `undefined` it seems to be okay when it's an empty array.